### PR TITLE
M57 release notes

### PR DESCRIPTION
**Pull request by infin8x**
This "release notes" post sums up the major changes in milestone 0.57.

### CODE REVIEW

File: themes/default/content/blog/pulumi-release-notes-0.57/index.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+draft: false
+meta_desc: "New in Pulumi: Amazon ECS Anywhere and App Runner, Sumo Logic support, Pulumi Packages updates, and easier updates for teammates in the Pulumi Service"
+meta_image: meta.png
+authors:
+    - alex-mullans
+tags:
+    - features
+---
+
+We release new versions of the Pulumi CLI every two weeks, new versions of the Pulumi Service multiple times a week, and new and updated cloud providers frequently. To keep up with all of these changes, you used to have to find the `CHANGELOG.md` files in the repo (or repos!) that you care about, which could get unwieldy. So, we're trying something new: a tri-weekly rollup of Pulumi news, features, and updates. If you find it useful, or have ideas on how we could improve it, let us know [on Twitter](https://twitter.com/pulumi) or in the [Pulumi Community Slack](https://slack.pulumi.com)!
```
Comment (lukehoban) [inline themes/default/content/blog/pulumi-release-notes-0.57/index.md:13]:
```suggestion
We release new versions of the Pulumi CLI every two weeks, new versions of the Pulumi Service multiple times a week, and new and updated cloud providers frequently. To keep up with all of these changes, you used to have to find the `CHANGELOG.md` files in the repo (or repos!) that you care about, which could get unwieldy. So, we're trying something new: a tri-weekly rollup of Pulumi release news, features, and updates. If you find it useful, or have ideas on how we could improve it, let us know [on Twitter](https://twitter.com/pulumicorp) or in the [Pulumi Community Slack](https://slack.pulumi.com)!
```

Also wonder if we can avoid the negative aspect of this phrasing?  Highlight that all realase notes are avialble in CHANGELOGs already today, but that we often want to highlight more details than we can in the CHANGELOG, and to highlight key new features across the dozens of repositories that make up the Pulumi open source project.

File: themes/default/content/blog/pulumi-release-notes-0.57/index.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
+---
+title: "Pulumi release notes: Milestone 58"
```
Comment (lukehoban) [inline themes/default/content/blog/pulumi-release-notes-0.57/index.md:2]: If we are going to frame this is being about a "milestone", we likely should talk about what milestones are - and point at where to understand more about them - possibly by pointing to https://github.com/pulumi/pulumi/wiki/Roadmap which talks about milestones in the intro, and updating the recent milestones to match what is in these blog posts?

That said - I'm not sure we *want* to focus on the inside baseball of milestones.  Perhaps just "What's New in Pulumi: June 9th, 2021"? or something along those lines?  That gives us flexibility to *not* tie this too strictly to milestones going forward if they aren't the right cadence.

File: themes/default/content/blog/pulumi-release-notes-0.57/index.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+  - Faster resource docs
+  - Self-Hosted Pulumi Service now supports Azure Storage and Minio backends
+
+<!--more-->
+
+## New and updated cloud providers
+
+### Pulumi supports Amazon ECS Anywhere and Amazon App Runner
+
+Earlier this month, Amazon announced a [new version of their Elastic Container Service](https://aws.amazon.com/blogs/containers/introducing-amazon-ecs-anywhere/) that allows you to deploy containers to almost any server and a new fully-managed container-native service called [AWS App Runner](https://aws.amazon.com/blogs/containers/introducing-aws-app-runner/). Pulumi had full same-day support for both new services.
```
Comment (lukehoban) [inline themes/default/content/blog/pulumi-release-notes-0.57/index.md:38]:
```suggestion
Earlier this month, Amazon announced [ECS Anywhere](https://aws.amazon.com/blogs/containers/introducing-amazon-ecs-anywhere/), an extension of Amazon Elastic Container Services that allows you to deploy containers to almost any server and  [AWS App Runner](https://aws.amazon.com/blogs/containers/introducing-aws-app-runner/), a new fully-managed container application service. Pulumi had full same-day support for both new services.
```

File: themes/default/content/blog/pulumi-release-notes-0.57/index.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+Learn how to get started with [ECS Anywhere]({{< relref "/blog/ecs-anywhere-launch" >}}) and [App Runner]({{< relref "/blog/deploy-applications-with-aws-app-runner" >}}).
+
+### Pulumi supports Sumo Logic
+
+This milestone, we released a new provider for Sumo Logic. You can now manage Sumo Logic's big data and analytics services with Pulumi infrastructure-as-code.
```
Comment (lukehoban) [inline themes/default/content/blog/pulumi-release-notes-0.57/index.md:44]:
```suggestion
The new Sumo Logic provider for Pulumi allows you to manage Sumo Logic's big data and analytics services with Pulumi infrastructure-as-code.
```

Perhaps include the code example (with lang chooser) from https://www.pulumi.com/docs/intro/cloud-providers/sumologic/ inline here to liven this up a little (have some code)?

File: themes/default/content/blog/pulumi-release-notes-0.57/index.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+
+Get started in the [SumoLogic provider docs]({{< relref "/docs/intro/cloud-providers/sumologic" >}}).
+
+### 100 new resources in the Azure Native provider
+
+In this milestone, we shipped 3 new versions of the Azure Native provider (1.10.0, 1.9.0, and 1.8.0) that collectively added 100 more Azure resources that you can manage with the native provider. These include new analytics, machine learning, and security capabilities across the breadth of the Azure platform.
```
Comment (lukehoban) [inline themes/default/content/blog/pulumi-release-notes-0.57/index.md:50]:
```suggestion
We shipped 3 new versions of the Azure Native provider (1.10.0, 1.9.0, and 1.8.0) that collectively added 100 new resources that you can manage with the Azure Native provider. These include new analytics, machine learning, and security capabilities across the breadth of the Azure platform.
```

Perhaps note that this includes all the new features and services Microsoft launched for Microsoft Build?

File: themes/default/content/blog/pulumi-release-notes-0.57/index.md [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+
+[See the full list](https://github.com/pulumi/pulumi-azure-native/blob/v1.10.0/CHANGELOG.md)
+
+## Pulumi engine and core technologies
+
+In this milestone, we shipped Pulumi versions [3.3.0](https://github.com/pulumi/pulumi/releases/tag/v3.3.0), [3.3.1](https://github.com/pulumi/pulumi/releases/tag/v3.3.1), and [3.4.0](https://github.com/pulumi/pulumi/releases/tag/v3.4.0). This section highlights key changes in those versions; for the full list, click the links for each version to visit the changelog for that version.
```
Comment (lukehoban) [inline themes/default/content/blog/pulumi-release-notes-0.57/index.md:56]:
```suggestion
We shipped Pulumi versions [3.3.0](https://github.com/pulumi/pulumi/releases/tag/v3.3.0), [3.3.1](https://github.com/pulumi/pulumi/releases/tag/v3.3.1), and [3.4.0](https://github.com/pulumi/pulumi/releases/tag/v3.4.0). The full list of changes in each version is available in the linked changelogs.  Some of the most exciting new features from these releases are highlighted below.

```

File: themes/default/content/blog/pulumi-release-notes-0.57/index.md [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+
+### Pulumi Packages schema documentation now available
```
Comment (lukehoban) [inline themes/default/content/blog/pulumi-release-notes-0.57/index.md:58]: This is from a docs release, not a Pulumi SDK release, right?  (only important as this looks nested under the Pulumi SDK/CLI section?)

File: themes/default/content/blog/pulumi-release-notes-0.57/index.md [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+
+Pulumi Packages, released at [PulumiUP earlier this year]({{< relref "/blog/pulumiup-pulumi-packages-multi-language-components" >}}) enable you to share Pulumi components with your colleagues and with the Pulumi community. But previously, it could be difficult to [author a Pulumi Package]({{< relref "/docs/guides/pulumi-packages" >}}) because the schema you use to define your package wasn't fully documented. Now, comprehensive schema documentation is available [in the docs]({{< relref "/docs/guides/pulumi-packages" >}}).
```
Comment (lukehoban) [inline themes/default/content/blog/pulumi-release-notes-0.57/index.md:60]:
```suggestion
[Pulumi Packages]({{< relref "/docs/guides/pulumi-packages/" >}}), released at [PulumiUP earlier this year]({{< relref "/blog/pulumiup-pulumi-packages-multi-language-components" >}}) enable you to share Pulumi components with your colleagues and with the Pulumi community. A key part of Pulumi Packages is the Pulumi schema that defines each resource in a package.  We have now introduced new [documentation for Pulumi Package Schema]({{< relref "/docs/guides/pulumi-packages" >}}). 
```

File: themes/default/content/blog/pulumi-release-notes-0.57/index.md [hunk 9]
```diff
@@ -0,0 +1,114 @@
...
+
+[#6777](https://github.com/pulumi/pulumi/issues/6777)
+
+### Input parameters can now be marked as secret
+
+It's now possible to mark any input parameter as a secret, which can be helpful when you need to pass in secret data, like when creating a Kubernetes Secret resource. Pulumi will protect these input parameters by encrypting them in the state and obfuscating them in outputs.
+
+[pulumi/pulumi#7062](https://github.com/pulumi/pulumi/issues/7062)
+
+### Plugin install output now goes to `stderr`
```
Comment (lukehoban) [inline themes/default/content/blog/pulumi-release-notes-0.57/index.md:74]: We can leave this out - it's a bug fix almost noone will understand.

File: themes/default/content/blog/pulumi-release-notes-0.57/index.md [hunk 10]
```diff
@@ -0,0 +1,114 @@
...
+
+Previously, if you used the `--json` flag on commands like `pulumi preview`, you would see the output of the Pulumi plugin acquisition process, which would break the ability to parse the JSON output. Now, the output of that process is sent to `stderr`.
+
+[pulumi/pulumi#5747](https://github.com/pulumi/pulumi/issues/5747)
+
+### Additional safety checks during `pulumi import`
```
Comment (lukehoban) [inline themes/default/content/blog/pulumi-release-notes-0.57/index.md:80]: We can leave this out - it's a bugfix almost noone will understand.

File: themes/default/content/blog/pulumi-release-notes-0.57/index.md [hunk 11]
```diff
@@ -0,0 +1,114 @@
...
+
+Pulumi Packages, released at [PulumiUP earlier this year]({{< relref "/blog/pulumiup-pulumi-packages-multi-language-components" >}}) enable you to share Pulumi components with your colleagues and with the Pulumi community. But previously, it could be difficult to [author a Pulumi Package]({{< relref "/docs/guides/pulumi-packages" >}}) because the schema you use to define your package wasn't fully documented. Now, comprehensive schema documentation is available [in the docs]({{< relref "/docs/guides/pulumi-packages" >}}).
+
+### Pulumi Packages schemas can reference other packages' types
+
+For more complex Pulumi Package scenarios, you may want to be able to reference the types in one Pulumi Package from another Pulumi Package. Previously, this was only possible if you authored your Pulumi Package in Go; now, it's possible in all Pulumi Languages.
+
+[#6777](https://github.com/pulumi/pulumi/issues/6777)
+
+### Input parameters can now be marked as secret
```
Comment (lukehoban) [inline themes/default/content/blog/pulumi-release-notes-0.57/index.md:68]: Suggest we can combine the three sections above into a single section on improvements to Pulumi Packages.

File: themes/default/content/blog/pulumi-release-notes-0.57/index.md [hunk 12]
```diff
@@ -0,0 +1,114 @@
...
+
+## Pulumi Service and Pulumi.com
+
+### Easier acceptance of invites to organizations
+
+Previously, you could accept an invitation to join an organization even if you didn't have the required identity provider (say, a SAML identity) associated with your Pulumi account. This left you unable to access the organization to which you just accepted an invite. Now, the Pulumi Service will prompt you to connect the required identity provider to your existing Pulumi account so you can successfully accept the invitation and join the organization.
+
+![Screenshot of identity provider setup prompt](saml.png)
+
+### Faster resource docs
```
Comment (lukehoban) [inline themes/default/content/blog/pulumi-release-notes-0.57/index.md:94]: I think we can leave this out - it's a bug fix - and we aren't yet back to great.

File: themes/default/content/blog/pulumi-release-notes-0.57/index.md [hunk 13]
```diff
@@ -0,0 +1,114 @@
...
+
+[pulumi/docs#5959](https://github.com/pulumi/docs/issues/5959)
+
+### Self-Hosted Pulumi Service now supports Azure Storage and Minio backends
+
+TODO?
+
+## Other news
+
+### Less flaky tests in `pulumi-kubernetes`
```
Comment (lukehoban) [inline themes/default/content/blog/pulumi-release-notes-0.57/index.md:106]: We can leave this out.

File: themes/default/content/blog/pulumi-release-notes-0.57/index.md [hunk 14]
```diff
@@ -0,0 +1,114 @@
...
+
+We fixed several issues causing flaky test runs in [`pulumi/pulumi-kubernetes`](https://github.com/pulumi/pulumi-kubernetes): [#1574](https://github.com/pulumi/pulumi-kubernetes/issues/1574) and [#1579](https://github.com/pulumi/pulumi-kubernetes/issues/1579).
+
+## TODO
+
+Node.js Multi-Lang Perf (Chris)
+Python SDK Exp Perf Fix (Luke)
+Google Native Fixes (Mikhail)
```
Comment (lukehoban) [inline themes/default/content/blog/pulumi-release-notes-0.57/index.md:114]: I think we can leave these all out.

File: themes/default/content/blog/pulumi-release-notes-0.57/index.md (hunk 2 above)
Comment (stack72) [reply themes/default/content/blog/pulumi-release-notes-0.57/index.md:2]: YEah I 100% agree! We don't really use milestones outside of Pulumi internal tracking - we regularly make releases outside of them

File: themes/default/content/blog/pulumi-release-notes-0.57/index.md [hunk 15]
```diff
@@ -0,0 +1,114 @@
...
+    - features
+---
+
+We release new versions of the Pulumi CLI every two weeks, new versions of the Pulumi Service multiple times a week, and new and updated cloud providers frequently. To keep up with all of these changes, you used to have to find the `CHANGELOG.md` files in the repo (or repos!) that you care about, which could get unwieldy. So, we're trying something new: a tri-weekly rollup of Pulumi news, features, and updates. If you find it useful, or have ideas on how we could improve it, let us know [on Twitter](https://twitter.com/pulumi) or in the [Pulumi Community Slack](https://slack.pulumi.com)!
+
+In this update:
+
+- New and updated cloud providers
+  - Pulumi supports Amazon ECS Anywhere and Amazon App Runner
+  - Pulumi supports Sumo Logic
```
Comment (stack72) [inline themes/default/content/blog/pulumi-release-notes-0.57/index.md:19]: We should link to pulumi-sumologic here IMO

Comment (stack72) [reply themes/default/content/blog/pulumi-release-notes-0.57/index.md:19]: TL;DR are a great sound of links without needing to read the full post 🤷

File: themes/default/content/blog/pulumi-release-notes-0.57/index.md (hunk 3 above)
Comment (infin8x) [reply themes/default/content/blog/pulumi-release-notes-0.57/index.md:38]:
```suggestion
Earlier this month, Amazon announced [ECS Anywhere](https://aws.amazon.com/blogs/containers/introducing-amazon-ecs-anywhere/), an extension of Amazon Elastic Container Service that allows you to deploy containers to almost any server, and [AWS App Runner](https://aws.amazon.com/blogs/containers/introducing-aws-app-runner/), a new fully-managed container application service. Pulumi had full same-day support for both new services.
```

File: themes/default/content/blog/pulumi-release-notes-0.57/index.md (hunk 1 above)
Comment (infin8x) [reply themes/default/content/blog/pulumi-release-notes-0.57/index.md:13]: Updated, LMK what you think

File: themes/default/content/blog/pulumi-release-notes-0.57/index.md (hunk 15 above)
Comment (infin8x) [reply themes/default/content/blog/pulumi-release-notes-0.57/index.md]: I made this a table of contents so folks can jump directly into the part of the post that interests them.

**Comment by infin8x** [comment]
Thanks @lukehoban @stack72 for the feedback! Awaiting @gthuang's feedback as well.

File: themes/default/content/blog/pulumi-release-notes-m57/index.md [hunk 16]
```diff
@@ -0,0 +1,165 @@
...
+
+Get started in the [SumoLogic provider docs]({{< relref "/docs/intro/cloud-providers/sumologic" >}}).
+
+### 100 new resources in the Azure Native provider
+
+We shipped 3 new versions of the Azure Native provider (1.10.0, 1.9.0, and 1.8.0) that collectively added 100 new resources that you can manage with the Azure Native provider. These new provider versions give you full access to the new resources and services announced at [Microsoft Build 2021](https://news.microsoft.com/build-2021-book-of-news/), including new analytics, machine learning, and security capabilities across the Azure platform.
+
+[See the full list](https://github.com/pulumi/pulumi-azure-native/blob/v1.10.0/CHANGELOG.md)
+
+## Pulumi engine and core technologies
```
Comment (gthuang) [inline themes/default/content/blog/pulumi-release-notes-m57/index.md:137]: should we just call this the Pulumi CLI and core technologies?

File: themes/default/content/blog/pulumi-release-notes-m57/index.md [hunk 17]
```diff
@@ -0,0 +1,165 @@
...
+
+In this milestone, we shipped Pulumi versions [3.3.0](https://github.com/pulumi/pulumi/releases/tag/v3.3.0), [3.3.1](https://github.com/pulumi/pulumi/releases/tag/v3.3.1), and [3.4.0](https://github.com/pulumi/pulumi/releases/tag/v3.4.0). The full list of changes in each version is available in the linked changelog; read on to learn about the some of the biggest changes.
```
Comment (gthuang) [inline themes/default/content/blog/pulumi-release-notes-m57/index.md:139]:
```suggestion
In this milestone, we shipped Pulumi versions [3.3.0](https://github.com/pulumi/pulumi/releases/tag/v3.3.0), [3.3.1](https://github.com/pulumi/pulumi/releases/tag/v3.3.1), and [3.4.0](https://github.com/pulumi/pulumi/releases/tag/v3.4.0). The full list of changes in each version is available in the linked changelog; read on to learn about some of the biggest changes.
```

File: themes/default/content/blog/pulumi-release-notes-m57/index.md [hunk 18]
```diff
@@ -0,0 +1,165 @@
...
+
+### Input parameters can now be marked as secret
+
+It's now possible to mark any input parameter as a secret, which can be helpful when you need to pass in secret data, like when creating a Kubernetes Secret resource. Pulumi will protect these input parameters by encrypting them in the state and obfuscating them in outputs.
+
+[pulumi/pulumi#7062](https://github.com/pulumi/pulumi/issues/7062)
+
+### Pulumi Packages: schema documentation and enhancements
+
+[Pulumi Packages]({{< relref "/docs/guides/pulumi-packages/" >}}), released at [PulumiUP earlier this year]({{< relref "/blog/pulumiup-pulumi-packages-multi-language-components" >}}) enable you to share Pulumi components with your colleagues and with the Pulumi community. A key part of Pulumi Packages is the Pulumi schema that defines each resource in a package.  We have now introduced new [documentation for Pulumi Package Schema]({{< relref "/docs/guides/pulumi-packages" >}}).
```
Comment (gthuang) [inline themes/default/content/blog/pulumi-release-notes-m57/index.md:149]:
```suggestion
[Pulumi Packages]({{< relref "/docs/guides/pulumi-packages/" >}}), released at [PulumiUP earlier this year]({{< relref "/blog/pulumiup-pulumi-packages-multi-language-components" >}}) enable cloud infrastructure resource provisioning to be defined once, in your language of choice, and made available to users in all Pulumi languages. A key part of Pulumi Packages is the Pulumi schema that defines each resource in a package.  We have now introduced new [documentation for Pulumi Package Schema]({{< relref "/docs/guides/pulumi-packages" >}}).
```

File: themes/default/content/blog/pulumi-release-notes-m57/index.md [hunk 19]
```diff
@@ -0,0 +1,165 @@
...
+draft: false
+meta_desc: "New in Pulumi: Amazon ECS Anywhere and App Runner, Sumo Logic support, Pulumi Packages updates, and easier updates for teammates in the Pulumi Service"
+meta_image: meta.png
+authors:
+    - alex-mullans
+tags:
+    - features
+---
+
+We regularly add new value across the Pulumi ecosystem, with a release of the Pulumi CLI every two weeks, frequent releases of our [cloud providers]({{< relref "/docs/intro/cloud-providers" >}}), and regular updates to the Pulumi Service. You can read about most of those changes in the the `CHANGELOG.md` files in each open source repository. To highlight some of the most exciting releases—including, for the first time, updates to the Pulumi Service—we're trying something new: a regular release notes post for Pulumi release news, features, and updates. If you find it useful, or have ideas on how we could improve it, let us know [on Twitter](https://twitter.com/pulumicorp) or in the [Pulumi Community Slack](https://slack.pulumi.com)!
```
Comment (gthuang) [inline themes/default/content/blog/pulumi-release-notes-m57/index.md:13]:
```suggestion
We regularly ship updates across the Pulumi ecosystem, with a release of the Pulumi CLI every two weeks, frequent releases of our [cloud providers]({{< relref "/docs/intro/cloud-providers" >}}), and regular updates to the Pulumi Service. You can read about most of those changes in the the `CHANGELOG.md` files in each open source repository. To highlight some of the most exciting releases—including, for the first time, updates to the Pulumi Service—we're trying something new: a regular release notes post for Pulumi release news, features, and updates. If you find it useful, or have ideas on how we could improve it, let us know [on Twitter](https://twitter.com/pulumicorp) or in the [Pulumi Community Slack](https://slack.pulumi.com)!
```

File: themes/default/content/blog/pulumi-release-notes-m57/index.md [hunk 20]
```diff
@@ -0,0 +1,165 @@
...
+
+## Pulumi engine and core technologies
+
+In this milestone, we shipped Pulumi versions [3.3.0](https://github.com/pulumi/pulumi/releases/tag/v3.3.0), [3.3.1](https://github.com/pulumi/pulumi/releases/tag/v3.3.1), and [3.4.0](https://github.com/pulumi/pulumi/releases/tag/v3.4.0). The full list of changes in each version is available in the linked changelog; read on to learn about the some of the biggest changes.
+
+### Input parameters can now be marked as secret
+
+It's now possible to mark any input parameter as a secret, which can be helpful when you need to pass in secret data, like when creating a Kubernetes Secret resource. Pulumi will protect these input parameters by encrypting them in the state and obfuscating them in outputs.
+
+[pulumi/pulumi#7062](https://github.com/pulumi/pulumi/issues/7062)
```
Comment (gthuang) [inline themes/default/content/blog/pulumi-release-notes-m57/index.md:145]:
```suggestion
Learn more at [pulumi/pulumi#7062](https://github.com/pulumi/pulumi/issues/7062)
```

File: themes/default/content/blog/pulumi-release-notes-m57/index.md [hunk 21]
```diff
@@ -0,0 +1,165 @@
...
+
+### Pulumi Packages: schema documentation and enhancements
+
+[Pulumi Packages]({{< relref "/docs/guides/pulumi-packages/" >}}), released at [PulumiUP earlier this year]({{< relref "/blog/pulumiup-pulumi-packages-multi-language-components" >}}) enable you to share Pulumi components with your colleagues and with the Pulumi community. A key part of Pulumi Packages is the Pulumi schema that defines each resource in a package.  We have now introduced new [documentation for Pulumi Package Schema]({{< relref "/docs/guides/pulumi-packages" >}}).
+
+We've also updated the Pulumi Package schema to make it more powerful. For more complex Pulumi Package scenarios, you may want to be able to reference the types in one Pulumi Package from another Pulumi Package. Previously, this was only possible if you authored your Pulumi Package in Go; now, it's possible in all Pulumi Languages.
+
+[#6777](https://github.com/pulumi/pulumi/issues/6777)
```
Comment (gthuang) [inline themes/default/content/blog/pulumi-release-notes-m57/index.md:153]:
```suggestion
Learn more at [#6777](https://github.com/pulumi/pulumi/issues/6777)
```

File: themes/default/content/blog/pulumi-release-notes-m57/index.md (hunk 16 above)
Comment (infin8x) [reply themes/default/content/blog/pulumi-release-notes-m57/index.md:137]:
```suggestion
## Pulumi CLI and core technologies
```

File: themes/default/content/blog/pulumi-release-notes-m57/index.md (hunk 21 above)
Comment (infin8x) [reply themes/default/content/blog/pulumi-release-notes-m57/index.md:153]:
```suggestion
Learn more at [pulumi/pulumi#6777](https://github.com/pulumi/pulumi/issues/6777)
```

**Review by gthuang — APPROVED** [review]

**Pull request merged**